integration_test/pg/all_test.exs

```diff
@@ -1,5 +1,6 @@
 Code.require_file "../cases/lock.exs", __DIR__
 Code.require_file "../cases/migration.exs", __DIR__
 Code.require_file "../cases/repo.exs", __DIR__
+Code.require_file "../cases/preloads.exs", __DIR__
```

Comment:
@jquadrin you will need to add this line to your pull request once I merge this to master and you rebase.